docs/apache-airflow/concepts/dynamic-task-mapping.rst

```diff
@@ -180,17 +180,14 @@ It is possible to use ``partial`` and ``expand`` with classic style operators as
 Mapping over result of classic operators
 ----------------------------------------
 
-If you want to map over the result of a classic operator you will need to create an ``XComArg`` object manually.
+If you want to map over the result of a classic operator, you need to explicitly reference the *output*, instead of the operator itself.
```

Comment:
```suggestion
If you want to map over the result of a classic operator, you can explicitly reference the *output*, instead of the operator itself.
```
Nit. Technically you _could_ still use `XComArg()` as well but we're saying there is a better way.

Reply:
The *need to* part is contrast to just passing in the operator object (i.e. not add `.output`), which wouldn’t work, so *can* would be confusing to me as well. Maybe *should*?

Reply:
Ah I see. That's fair. Words are fun.